modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/UserManager.java

```diff
@@ -182,4 +182,10 @@ default void updateGroup(Group oldGroup, Group newGroup)
 
         throw new NotImplementedException();
     }
-}
+
+    default List<Attribute> getEnterpriseUserSchema() throws CharonException, NotImplementedException,
```

Comment:
shall we add doc comments on this new API

Reply:
Added